portfolio/src/main/webapp/script.js

```diff
@@ -28,7 +28,15 @@ function addRandomQuoteOrFact() {
 }
 
 function hello(){
-   fetch('/data').then(response=>response.text()).then((hi)=>{
+   fetch('/data').then(response=>response.json()).then((hi)=>{
     document.getElementById('response').innerText = hi;
   });
+}
+
+
+/** Creates an <li> element containing text. */
+function createListElement(text) {
```

Comment:
I haven't seen you used this method - it will help you in displaying the comments as a list, instead of one long string.